tests/e2e_tests/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/eslint-recommended", "plugin:@typescript-eslint/recommended"],
+  "rules": {
+    "prettier/prettier": 2,
+    "consistent-this": ["error", "self"],
+    "import/no-named-as-default": 0,
+    "no-console": 0,
+    "no-multiple-empty-lines": 2,
+    "no-prototype-builtins": 0,
+    "quotes": [2, "single", { "allowTemplateLiterals": true }],
```

Comment:
quotes rule doe snot have three options, only two, what does `2` do here?

Reply:
do we consider `avoidEscape` ?

Reply:
hmm looks like the number of options has changed in one of the previous version... we need escapes for some of the texts in the GUI...

Reply:
ah no, the options are counted after the severity 🤦 - so all good here 👌